Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -24,5 +24,7 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
 
-    // Add your necessary variables here
+    double current_;
```

Comment:
I would name this variable more explicitly, besides the first instance, this value is always a past value.